zdm/beams.py

```diff
@@ -48,15 +55,8 @@ def load_beam(suffix,basedir=beams_path):
 	if logb.size == omega_b.size+1:
 		# adjust for the bin centre
 		db=logb[1]-logb[0]
-		logb=logb[:-1]+db
+		logb=logb[:-1]+db/2.
```

Comment:
minor bug fix: logb should be in the centre of the bin, this made it at the top of the bin. This bug will have resulted in artificially increased sensitivity by approximately 1%.

Reply:
good catch